.github/workflows/reusable-docs.yml

```diff
@@ -22,12 +22,14 @@ jobs:
     env:
       branch_base: 'origin/${{ github.event.pull_request.base.ref }}'
       branch_pr: 'origin/${{ github.event.pull_request.head.ref }}'
+      commits: ${{ github.event.pull_request.commits }}
       refspec_base: '+${{ github.event.pull_request.base.sha }}:remotes/origin/${{ github.event.pull_request.base.ref }}'
       refspec_pr: '+${{ github.event.pull_request.head.sha }}:remotes/origin/${{ github.event.pull_request.head.ref }}'
     steps:
     - name: 'Check out latest PR branch commit'
       uses: actions/checkout@v4
       with:
+        persist-credentials: false
```

Comment:
Same here, there's a fetch in the next step -- or am I misunderstanding what `persist-credentials` is for?
(On a related note, if `persist-credential` is rarely needed and a security risk, I wonder if it should default to `false` and require an explicit `true` when needed in the next release of `actions/checkout`, so that we don't have to add it everywhere.)